tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -31,6 +31,7 @@
   private TileRenderTask mLastRunTileRenderTask;
 
   private DetailLevel mDetailLevelToRender;
+  private DetailLevel mLastRequestedDetailLevel;
```

Comment:
what's the rationale behind separating `mLastRequestedDetailLevel ` and `mLastRenderedDetailLevel`?